ivy/data_classes/array/experimental/linear_algebra.py

```diff
@@ -675,3 +675,55 @@ def tucker(
             tol=tol,
             verbose=verbose,
         )
+
+    def tt_matrix_to_tensor(
+        self: Union[ivy.Array, ivy.NativeArray],
+        /,
+        *,
+        out: Optional[Union[ivy.Array, ivy.NativeArray]] = None,
```

Comment:
```suggestion
        out: Optional[ivy.Array] = None,
```
out will always be an `ivy.Array`